app/konnect/runtime-manager/composite-runtime-groups/index.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Composite Runtime Groups
+content_type: explanation
+badge: enterprise
+---
+
+A composite runtime group is a read-only group that combines configuration from
+its members, which are standard runtime groups. All of the standard runtime groups within a 
+composite runtime group share the same cluster of runtime instances. 
+
+## Composite vs standard groups
+
+In a standard runtime group setup, each team configures and manages their own runtime instances.
+For example, in the following diagram Team Blue configures Runtime Group Blue, which then uses a set of runtime instances that only run Blue configuration; the same happens with Team Yellow.
+
+![Standard runtime group](/assets/images/docs/konnect/konnect-standard-rg.svg)
+> _**Figure 1:** Standard runtime group workflow_
+
+In a composite runtime group setup, each team still administers their own runtime group, but the runtime instances are shared. 
+
+The following diagram illustrates using a composite runtime group for a federated platform administrator model. In this example, Team Blue configures Runtime Group Blue, which is then combined with the configuration from Team Yellow. In addition, the composite runtime group contains Runtime Group Green, which is managed by a central platform team. The central team provides global plugin configuration, which is added to any configuration that teams Blue and Yellow provide.
+
+The runtime instances in the cluster use the combined configuration from all three groups.
+
+![Composite runtime group](/assets/images/docs/konnect/konnect-composite-rg.svg)
+> _**Figure 2:** Composite runtime group workflow_
+
+A composite runtime group can contain up to 256 runtime groups. 
+You can add or remove up to 50 member runtime groups at a time.
+
+Each standard group can be a member of no more than 5 composite runtime groups.
+
+## Runtime instances 
+
+In a composite runtime group, the composite configuration from all member runtime groups is pushed to each runtime instance.
+
+A runtime instance (also known as a {{site.base_gateway}} data plane node) can only connect to a single runtime group in a cluster.
+This means that in a composite runtime group, all runtime instances must be managed from the composite group itself. 
+Members of a composite group can't have their own runtime instances. 
+
+When adding a standard group to a composite, make sure it has no connected runtime instances.
+
+The runtime instances of a composite runtime group are not visible to a member runtime group.
+
+## Configuring core entities
+
+There are some special cases and behaviors to note for core entities in a composite runtime group.
+
+All entities in a composite runtime group must have unique names and IDs. 
+For example, if two members of a composite runtime group both have a service named `example_service`, 
+it will cause an [invariance](/konnect/runtime-manager/composite-runtime-groups/invariances/) which must be resolved to restore function.
```

Comment:
```suggestion
it will cause a [conflict](/konnect/runtime-manager/composite-runtime-groups/conflicts/) which must be resolved to restore function.
```